packages/multichain/CHANGELOG.md

```diff
@@ -7,6 +7,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0]
+
+### Added
+
+- Add key Multichain API methods ([#4813](https://github.com/MetaMask/core/pull/4813))
+  - Adds `getInternalScopesObject` and `getSessionScopes` helpers for transforming between `NormalizedScopesObject` and `InternalScopesObject`.
+  - Adds `caipPermissionAdapterMiddleware` for enforcing CAIP-25 permission on the EIP-1193 API.
+  - Adds `walletGetSession`, `walletInvokeMethod`, and `walletRevokeSession` handlers.
```

Comment:
Should these be underscored?

Reply:
done here: https://github.com/MetaMask/core/pull/5184/commits/720eb37a1b0ffff0890b3724129d8bb367023e99